android/CouchbaseLite/src/androidTest/java/com/couchbase/lite/ArrayTest.java

```diff
@@ -716,9 +716,8 @@ public void validate(Array a) {
                 assertEquals(1.49F, a.getFloat(1), 0.0F);
                 assertEquals(1.49, a.getDouble(1), 0.0);
 
-                // TODO:
-//                assertEquals(1.50, a.getObject(2));
-//                assertEquals(1.50, a.getNumber(2));
+                assertEquals(1.50, ((Number)a.getObject(2)).doubleValue(), 0.0);
+                assertEquals(1.50, a.getNumber(2).doubleValue(), 0.0);
```

Comment:
same reason with above. Some of the Double value is stored as Float.
1.50 on java is Double, and a.getObject() returns Float as Fleece store it as Float.